impl/src/test/resources/privateKey.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
My pref is to generate this at test time, so you don't need to:
* Have a key that could expire
* run into code scanning issues that find a "private key" checked into a repository

NOTE: this isn't always possible to avoid, but when it is we should try

Reply:
noted.

Reply:
Removed the privateKey from version control.